CONTRIBUTING.md

```diff
@@ -24,6 +24,7 @@
 
 - Close and relaunch Ubuntu to source your `~/.bashrc`.
 - [Install Python's build dependencies](https://github.com/pyenv/pyenv/wiki#suggested-build-environment).
+- If `.tool-versions` contains `ruby`, [install Ruby's build dependencies](https://github.com/rbenv/ruby-build/wiki#suggested-build-environment).
```

Comment:
Nit: wrap long line at 80 columns for consistency/readability.

Reply:
I put the line break in the place that seemed most logical to me that was close to 80 columns. Let me know if I should change its placement.

Reply:
Oh, the line break should go before the `[` so that there's not an extra space before the period in the rendered Markdown.